### DISCUSSION TITLE
Custom auth tab not loading!

### DISCUSSION THREAD

**Discussion by kishornlstech** [Web3Auth/Web3Auth#1207] [Help]
All other tabs opening where i can create project or setings but create verifier page is just blank. Also my app login page is blank too. Login page opens but navigates to black page(beta.openlogin.com). Everything was fine yesterday.

**Comment by BboyStatix** [comment]
Hi, would it be possible to provide us with some additional context? Perhaps some screenshots or a screen recording?

**Comment by kishornlstech** [comment]
Hello Team,
PFA......Nothing is loading on dashboard thats reason our Web3Auth in
mobile app is not working.

Thanks and regards,
Kishor

**Reply by BboyStatix** [reply]
Are you facing any network errors? The stuff at https://dashboard.web3auth.io/ seems to be loading just fine for me.

**Comment by kishornlstech** [comment]
Hello ,
Not facing any issue on network side it’s not working in 2 different
systems.. for me

**Reply by BboyStatix** [reply]
More information would be helpful so I can debug the issue.

**Reply by kishornlstech** [reply]
https://user-images.githubusercontent.com/118105181/212607859-5ca09ec3-497a-48c4-91da-dc375f281df5.mp4

**Comment by kishornlstech** [comment]
This web page  is not loading that’s issue

**Comment by kishornlstech** [comment]
We have uploaded one video for reference

**Reply by BboyStatix** [reply]
Do you also have console logs that you could provide?

**Comment by kishornlstech** [comment]
No console logs I don’t have .

**Comment by BboyStatix** [comment]
@kishornlstech can you try again today to see if the problem has been resolved?

**Comment by kishornlstech** [comment]
It’s working
